how-to/2_Geometry/CreateCustomGeometry.ipynb

```diff
@@ -0,0 +1,410 @@
+    "from zipfile import ZipFile\n",
+    "import numpy as np"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Download and extract the file `SparseBeads_B12_L1` from Zenodo [here](https://zenodo.org/records/290117/files/SparseBeads_B12_L1.zip)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "--2024-10-19 16:39:46--  https://zenodo.org/records/290117/files/SparseBeads_B12_L1.zip\n",
+      "Resolving zenodo.org (zenodo.org)... 188.184.98.238, 188.184.103.159, 188.185.79.172, ...\n",
+      "Connecting to zenodo.org (zenodo.org)|188.184.98.238|:443... connected.\n",
+      "HTTP request sent, awaiting response... 200 OK\n",
+      "Length: 20588556 (20M) [application/octet-stream]\n",
+      "Saving to: ‘/home/efv97572/datasets/SparseBeads_B12_L1.zip.1’\n",
+      "\n",
+      "SparseBeads_B12_L1. 100%[===================>]  19.63M  52.6MB/s    in 0.4s    \n",
+      "\n",
+      "2024-10-19 16:39:47 (52.6 MB/s) - ‘/home/efv97572/datasets/SparseBeads_B12_L1.zip.1’ saved [20588556/20588556]\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "!wget -P ~/datasets https://zenodo.org/records/290117/files/SparseBeads_B12_L1.zip\n",
+    "\n",
+    "z = ZipFile(os.path.expanduser(\"~/datasets/SparseBeads_B12_L1.zip\"))\n",
+    "z.extractall(os.path.expanduser(\"~/datasets\"))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First let's load the raw data file and look at its properties:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[[60369. 60004. 60659. ... 60414. 61058. 60764.]\n",
+      " [60274. 60342. 60387. ... 60143. 60245. 60447.]\n",
+      " [59631. 60254. 60189. ... 60455. 59983. 60137.]\n",
+      " ...\n",
+      " [62839. 62503. 62314. ... 62739. 62601. 63342.]\n",
+      " [62693. 62364. 61960. ... 63250. 62819. 62583.]\n",
+      " [62151. 63011. 62955. ... 62714. 62797. 62754.]] \n",
+      " (2520, 2000)\n"
+     ]
+    }
+   ],
+   "source": [
+    "file_name = '/home/efv97572/datasets/SparseBeads_B12_L1/CentreSlice/Sinograms/SparseBeads_B12_L1_0001.tif'\n",
+    "data_reader = TIFFStackReader(file_name)\n",
+    "data = data_reader.read()\n",
+    "\n",
+    "print(data, \"\\n\", data.shape)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can see the data is an array of pixel intensities of 2520 projections, containing 2000 horizontal pixels and a single vertical slice.  \n",
+    "\n",
+    "More detailed metadata for this experiment is stored in the file `SparseBeads_B12_L1.xtek2dct`. We can read the contents of this file, and get the needed information to create the `AcquisitionGeometry`:"
```

Comment:
I think the advantage of this notebook compared to the where_is_my_reader exercise is that it's shorter and simpler for people to understand how to create a geometry quickly. I would maybe skip this part of reading the .xtek2dct file and instead just say "we know the Source To Object distance = 121.932688713074 and Source To Detector distance = 1400.207". This also makes it more generic for data from other systems not just Nikon

Reply:
Done!